decoimpact/data/parsers/parser_filter_extremes_rule.py

```diff
@@ -41,7 +44,8 @@ def parse_dict(self, dictionary: Dict[str, Any], logger: ILogger) -> IRuleData:
         ]
         output_variable_name: str = get_dict_element("output_variable", dictionary)
         description: str = get_dict_element("description", dictionary, False) or ""
-        extreme_type: str = get_dict_element("extreme_type", dictionary)
+        extreme_type: str = get_dict_element("extreme_type", dictionary) or "peaks"
```

Comment:
but now you set the default to peaks. Is that what the PO wants?

Reply:
good question, etreme_type is a required field, so you can not leave it empty. Setting a default does not make sense. I'll remove it